src/node/rpc/metrics.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#include "ds/histogram.h"
+#include "ds/logger.h"
+
+#include <nlohmann/json.hpp>
+
+#define HIST_MAX (1 << 17)
+#define HIST_MIN 1
+#define HIST_BUCKET_GRANULARITY 5
+#define TX_RATES 4000
```

Comment:
That looks like the number of transaction rate buckets? TX_RATE_BUCKETS_LEN or something like that?